filebeat/module/mongodb/log/ingest/pipeline.json

```diff
@@ -0,0 +1,41 @@
+{
+  "description": "Pipeline for parsing MongoDB log logs",
```

Comment:
Mongodb log logs -> I think I start to understand why you call the filset `log`. What exactly is in these logs?